### PR TITLE
Add SHRUB, SBR, MSTR, SIGMA, FWOG, LESTER, & HAPPY token price

### PR DESCRIPTION
**Pull request by lydiapuspita**
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

[...]

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql [hunk 1]
```diff
@@ -707,5 +707,8 @@ FROM
         ('moodeng-moo-deng-moodengsolcom', 'solana', 'MOODENG', 'ED5nyyWEzpPPiWimP8vYm7sD7TD3LAt3Q3gRTWHzPJBY', 6),
         ('scf-smoking-chicken-fish', 'solana', 'SCF', 'GiG7Hr61RVm4CSUxJmgiCoySFQtdiwxtqf64MsRppump', 6),
         ('luna-luna-by-virtuals', 'solana', 'LUNA', '9se6kma7LeGcQWyRBNcYzyxZPE3r9t9qWZ8SnjnN3jJ7', 8),
-        ('goat-goatseus-maximus', 'solana', 'GOAT', 'CzLSujWBLFsSjncfkh59rUFqvafWcY5tzedWJSuypump', 6)
+        ('goat-goatseus-maximus', 'solana', 'GOAT', 'CzLSujWBLFsSjncfkh59rUFqvafWcY5tzedWJSuypump', 6),
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql:710]: https://solscan.io/token/CzLSujWBLFsSjncfkh59rUFqvafWcY5tzedWJSuypump

File: dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql [hunk 2]
```diff
@@ -707,5 +707,8 @@ FROM
...
+        ('fwog-fwog', 'solana', 'FWOG', 'A8C3xuqscfmyLrte3VmTqrAq8kgMASius9AFNANwpump', 6),
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql:711]: https://solscan.io/token/A8C3xuqscfmyLrte3VmTqrAq8kgMASius9AFNANwpump

File: dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql [hunk 3]
```diff
@@ -707,5 +707,8 @@ FROM
...
+        ('lester-litecoin-mascot', 'solana', 'LESTER', '5z3iCe53hUANTiG8Js8RjHNE2Arjik7L2CXLyr2rpump', 6),
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql:712]: https://solscan.io/token/5z3iCe53hUANTiG8Js8RjHNE2Arjik7L2CXLyr2rpump

File: dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql [hunk 4]
```diff
@@ -707,5 +707,8 @@ FROM
...
+        ('happy-happycat', 'solana', 'HAPPY', 'HAPPYwgFcjEJDzRtfWE6tiHE9zGdzpNky2FvjPHsvvGZ', 9)
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql:713]: https://solscan.io/token/HAPPYwgFcjEJDzRtfWE6tiHE9zGdzpNky2FvjPHsvvGZ

**Review by Hosuke — APPROVED** [review]
`token_id`s are correct.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**